verifier.py

```diff
@@ -369,72 +429,213 @@ def __init__(self):
             '#fileBrowser > div.db-infobar > div > div': '#fileBrowser > div.db-infobar > div > div',  # Project preview
             '#tb-tbody': '#fileBrowser > div.db-main > div.db-non-load-template.m-md.p-md.osf-box'  # Project browser
         }
-        self.harvest_pages(run_info['institution_urls'], '', InstitutionDashboardPage)
+        # self.harvest_pages(run_info['institution_urls'], '', InstitutionDashboardPage)
+        # self.size_comparison()
+        # self.spot_check()
+
+    def run_verifier(self, json_filename, json_list):
+        self.harvest_pages(json_filename, json_list, '', InstitutionDashboardPage)
         self.size_comparison()
         self.spot_check()
 
 
-def main():
-    # Identifying problems
-
-    # for modularization and then concatenate lists
+# first verification and then scraping the failed pages
+def initial_verification(json_file, num_retries: object) -> object:
+    with codecs.open(json_file, mode='r', encoding='utf-8') as failure_file:
```

Comment:
Can you replace failure_file with something like task_file to not misrepresent it?